components/director/cmd/jwtgenerator/main.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"flag"
+	"fmt"
+
+	"github.com/form3tech-oss/jwt-go"
+)
+
+type Claims struct {
+	Scopes string `json:"scopes"`
+	Tenant string `json:"tenant"`
+	jwt.StandardClaims
+}
+
+func main() {
```

Comment:
do you really need a whole go main file for that ? cant this be just a shell script ?